ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -640,6 +640,17 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: vsphere-8-ipi-private-proxy-f14-disasterrecovery
+  cron: 11 11 22 * *
```

Comment:
The test name is `f14`, while the cron entry indicates its f28. Perhaps make them in sync ?

Reply:
Yes, notice this wrong setting.

Reply:
Updated.